src/main/resources/static/js/patients.js

```
@@ -0,0 +1,284 @@
+    if (!patientsContainer || !emptyState) return;
+
+    const patientCards = patientsContainer.querySelectorAll('.patient-card');
+    let visibleCount = 0;
+
+    // Loop through each patient card and check if it matches search term
+    patientCards.forEach(card => {
+        let matchFound = false;
+
+        // Check patient name
+        const patientNameEl = card.querySelector('.patient-name');
+        if (patientNameEl && patientNameEl.textContent.toLowerCase().includes(searchTerm)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)